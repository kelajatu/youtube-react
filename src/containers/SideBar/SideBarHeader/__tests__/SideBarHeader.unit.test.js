import React from "react";
import { SideBarHeader } from "../SideBarHeader";
import { shallow } from "enzyme";

describe("SideBarHeader", () => {
  it("renders", () => {
    const wrapper = shallow(<SideBarHeader />);
    expect(wrapper).toMatchSnapshot();
  });
  it("renders with empty title", () => {
    const wrapper = shallow(<SideBarHeader title="" />);
    expect(wrapper).toMatchSnapshot();
  });
  it("renders with title", () => {
    const wrapper = shallow(<SideBarHeader title="Title" />);
    expect(wrapper).toMatchSnapshot();
  });
});
