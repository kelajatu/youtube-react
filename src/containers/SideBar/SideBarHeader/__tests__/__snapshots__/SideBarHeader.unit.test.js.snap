// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SideBarHeader renders 1`] = `
<MenuItem>
  <MenuHeader
    className="side-bar-header"
  />
</MenuItem>
`;

exports[`SideBarHeader renders with empty title 1`] = `
<MenuItem>
  <MenuHeader
    className="side-bar-header"
  />
</MenuItem>
`;

exports[`SideBarHeader renders with title 1`] = `
<MenuItem>
  <MenuHeader
    className="side-bar-header"
  >
    TITLE
  </MenuHeader>
</MenuItem>
`;
